webui.py

```diff
@@ -125,55 +117,58 @@ def parse_args():
                 chatbot = gr.Chatbot(elem_id="chatbot-container")
                 msg = gr.Textbox(label="Prompt")
                 stop = gr.Button("Stop")
-    llm_chain, llm = init_chain(model, tokenizer)
+
+    llm_chain, llm = init_llm_chain("./src/quantized_model\stabilityai__stable-code-instruct-3b.gguf")
```

Comment:
this path should be dynamic, based on what repo id user has added, return converted models path from quantize.py